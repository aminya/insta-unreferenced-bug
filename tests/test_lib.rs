use insta_bug::something;
use insta::assert_snapshot;

#[test]
fn test_lib() {
    assert_snapshot!(something());
}