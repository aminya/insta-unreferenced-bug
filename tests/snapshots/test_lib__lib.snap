---
source: tests/test_lib.rs
expression: something()
---
Hello, world!
